src/pretix/control/templates/pretixcontrol/waitinglist/index.html

```diff
@@ -261,9 +264,17 @@ <h1>{% trans "Waiting list" %}</h1>
         {% if "can_change_orders" in request.eventpermset %}
             <div class="batch-select-actions">
                 <button type="submit" class="btn btn-danger btn-save" name="action" value="delete">
-                    <i class="fa fa-trash"></i>{% trans "Delete selected" %}
+                    <i class="fa fa-trash"></i>
+                    {% trans "Delete selected" %}
+                </button>
+                {% if request.event.has_subevents %}
+                <button type="submit" class="btn btn-primary btn-save" name="action" value="update">
+                    <i class="fa fa-share-square-o" aria-hidden="true"></i>
+                    {% trans "Transfer selected" %}
```

Comment:
The wording in this PR is very unclear. Is it an "update" or a "transfer" what we're doing here?

Reply:
Sorry, this button should not be here as this was part of Ben’s attempt for batch-transfer. I will remove it. But yes, action should have been „transfer“.

Reply:
Regarding `update` vs. `transfer` – should the URL have the parameter called /transfer/ instead of /update/ as well? And probably the View-Class should be called `EntryTransfer` instead of `EntryUpdate`?

Reply:
Changed wording (as well as ClassNames and URLs) from „update“ to „transfer“. Makes way more sense. Thanks for pointing that out!